src/CollapsibleCard/index.stories.js

```diff
@@ -179,6 +179,28 @@ WithError.args = {
   ),
 };
 
+export const WithCustomTitle = Template.bind({});
+WithCustomTitle.args = {
+  renderTitle: (isOpen) => (
+    <div className="padding--all">
+      This card is <strong>{isOpen ? "open" : "closed"}</strong>.
+    </div>
+  ),
```

Comment:
This is what gives us unlimited flexibility moving forward. Collapsible cards are used in a lot of our new designs, and we have a new designer joining soon too. Keep this component flexible will keep it alive and useful as we try out different stuff in design